modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm

```diff
@@ -65,7 +64,7 @@
 	alt_titles = list("Curator", "Librarian", "Journalist", "Archivist")
 
 /datum/job/customs_agent
-
+	alt_titles = list("Customs Agent", "Customs Officer", "Cargo Watchman", "Cargo Officer")
```

Comment:
I'd rather keep clear of alt-titles including "officer" or other such sec-like terms. We really don't want confusion.

Reply:
I'd only really want Cargo Guard as an alt title for this.